### ISSUE TITLE
Mod Manager issue?  Maybe?

### ISSUE THREAD

**Issue by MechanosG** [O-n-y/oxy.mods#31]
This is kind of a long story, so bear with me.

Alright so..  today a mod called [No Manual Delivery](https://steamcommunity.com/sharedfiles/filedetails/?id=2047308624&tscn=1603679304) updated.  When I launched ONI, Mod Manager detected that it didn't update properly, so it said I needed to restart.  So I restarted ONI.  The 2nd time loading it popped up an error involving version mismatch, and said I needed to restart again.  So I restarted again.  3rd time loading, Mod Manager was happy and said everything was up to date.  But upon loading a newish game save, my game refused to unpause, CPU usage spiked, and eventually my GUI disappeared.  I ended up having to end task ONI.

At this point I assumed the No Manual Delivery update must be broken, so I disabled the mod and tested loading my save game.  It loaded fine.  So I went and reported the bug in the comment section of No Manual Delivery.  The author responded, blaming Mod Manager, and telling me to go use Mod Updater instead, even though Mod Manager seems to be working fine for all my other mods.  What do you think is going on?

**Comment by O-n-y** [comment]
You can check which mod causing your game to crash or getting an error - by simply disabling it.
So if you have an error after some mod updated (you can check which exact version of the mod you run in the mods list), you can clearly say which gives the error. 
And even better, you can disable all other mods and leave enabled the only one which is under suspicion. 

Of course, some mods can be not compatible with each other, but this happens only if those do patch (change) the same basic functions, which in this case unlikely, as Mod Manager does not modify game functions that influence gameplay in any sense. 

Mod Manager does not have any other functionality other than updating mods and other similar connected to Mods List.

Also, the last version of Mod Manager does not know any known crashed reported and working stable.

So I would say:
1. It can be an issue in the version of that mod you are using
2. It can have an incompatibility with some other mod you have, which do patch similar game functions (unlikely it is a Mod Manager tho)

**Comment by MechanosG** [comment]
Well, as I said, I did disable it, and the problem went away.  But he's more or less claiming that if I stop using mod manager and instead use mod updater, his mod will magically start working.  And he's disregarding everything I post, saying I haven't tried his "fix" yet.  I haven't tested using only his mod, because he's more or less accusing the mod of not being installed correctly due to mod manager.  So other than trying this proposed fix of disabling mod manager, installing mod updater, and re-enabling his mod just to rule out if it changes anything, I'm not sure how to proceed other than just not using his mod anymore lol.  It sounds like you're confident that mod manager didn't cause this, so I guess all I can do is try his fix, and if it doesn't fix the problem, maybe he'll look into the issue.

**Comment by O-n-y** [comment]
The is no such thing a "the mod of not being installed correctly", as all mods just a zip archive and that's all, there is no installation really, other than unzipping in a folder.
Also, there never was a report about Mod Manager "installing not correctly" from all-time of its work.

But surely you can try to use any mods you like :-)

**Comment by MechanosG** [comment]
Hmm.  Strangely enough, there does seem to be a weird issue between Mod Manager and No Manual Delivery.  I disabled No Manual Delivery because I just didn't want to bother with it anymore  but haven't unsubscribed from it.  Every 1 or 2 times I launch ONI, Mod Manager pops up an update message again, claiming a version check failed on No Manual Delivery, asking me to update it and restart.  I've updated it 3 times now despite there being no new updates since the 25th.  So..  surprisingly there does seem to be something weird going on.  Only his mod seems to be having version check problems, and I have like.. 43 installed or something.

**Comment by O-n-y** [comment]
Can be it is just not updating no matter what, you can check which version the author claims as last and which you can see in the mods list.
Never the less it has nothing with incompatibility issue.

**Comment by MechanosG** [comment]
I asked the mod author.  He says the version should be 1.1.7603.35871.  Starting up ONI just now, Mod Manager again popped up a message saying version check failed, and wants to update No Manual Delivery.  Clicking cancel and going into the mod menu, it says it's already at version 1.1.7603.35871, yet also says it's outdated.
![20201029113307_1](https://user-images.githubusercontent.com/4360512/97617351-e4f8af00-19da-11eb-9cc2-3f3ecd529446.jpg)

**Comment by O-n-y** [comment]
@MechanosG, this can be a timestamp check issue (a few hours difference), hover over outdated it will show technical information, then screenshot it.
Which version of Mod Manager you have installed?

p.s. in any case this has nothing to do with how a particular mod (in your case "No Manual Delivery") works.

**Comment by MechanosG** [comment]
2.9.6.221 according to the mod manager list.

**Comment by O-n-y** [comment]
hover over outdated it will show technical information, then screenshot it

**Comment by MechanosG** [comment]
Something bizarre seems to have happened.  I opened ONI to grab a screenshot, and I got a message saying No Manual Delivery was uninstalled.  I didn't uninstall it though and don't know what triggered this message.  Confused, I restarted ONI, and....  No Manual Delivery still shows up in my mod list disabled, but now it's at the bottom of the load order.  I tried enabling and disabling it, and everything seemed normal.  The update problem usually came back after 24 hours so I waited a day then launched ONI again.  No update problem this time.  It's as if the problem solved itself.  But I really dunno what happened.  I guess at least for now, the update problem might've fixed itself.
